modules/publisher/views.py

```diff
@@ -221,6 +221,15 @@ def post_market_snap(snap_name):
         images_files = []
         images_json = None
 
+        # Add existing screenshots
+        screenshots_uploaded = api.snap_screenshots(flask.request.form['snap_id'])
+        state_screenshots = loads(flask.request.form['state'])['images']
+        for screenshot in state_screenshots:
+            for screenshot_uploaded in screenshots_uploaded:
+                if screenshot['url'] == screenshot_uploaded['url']:
```

Comment:
So you are not using `status` flag for now?

Reply:
no finally I did not because I need to catch more data from the binary-metadata endpoint (screenshot endpoint).
And I manage to get the info I need without the status flag